gloo/docs/installation/gateway/docker-compose-consul/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Run Gloo Gateway Locally with Hashicorp Consul & Vault
```

Comment:
Not sure about this, but I tend to associate the term " running locally" with something to run in development mode. We have been talking about "running in VMs", could that be a better term?

Reply:
doesn't have anything to do with a vm. wherever this is run, it's the local environment (processes run on the same host where `docker-compose` is run)